src/main/java/uk/nhs/hee/tis/trainee/sync/event/CurriculumEventListener.java

```diff
@@ -53,7 +72,25 @@ public void onAfterSave(AfterSaveEvent<Curriculum> event) {
 
   @Override
   public void onAfterDelete(AfterDeleteEvent<Curriculum> event) {
-    // TODO: Implement.
     super.onAfterDelete(event);
+
+    String curriculumId = event.getSource().getString("_id");
+    sendProgrammeMembershipMessages(curriculumId, Operation.DELETE);
```

Comment:
I'm not sure what we should actually do here.
For placements I've avoided deleting the placement from the trainee profile for "upstream" data changes, only the placement being deleted will affect the profile.

The only deletes I've cascaded are those that don't directly impact the trainee profile, i.e. deleting from the intermediate lookup collections in the sync db.

Reply:
Ah ok, sorry, I had the wrong-end of the stick. Its a much simpler PR now 😁